Content.Shared/Weapons/Reflect/ReflectComponent.cs

```diff
@@ -39,6 +39,22 @@ public sealed partial class ReflectComponent : Component
     [DataField]
     public bool Innate = false;
 
+    // WD START
+    /// <summary>
+    /// If the item for reflection needed in inventory slots only - select Body.
+    /// If the item for reflection needed in hands only - select Hands.
+    /// Otherwise it will reflect in any inventory position.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite), DataField("howToUse")]
```

Comment:
```suggestion
    [ViewVariables(VVAccess.ReadWrite), DataField]
```
Не нужно указывать строковое название, если оно совпадает с названием переменной с маленькой буквы